libs/datamodel/core/src/transform/ast_to_dml/validate.rs

```diff
@@ -218,29 +220,6 @@ impl<'a> Validator<'a> {
         ast_model: &ast::Model,
         model: &dml::Model,
     ) -> Result<(), DatamodelError> {
-        let multiple_single_field_id_error = Err(DatamodelError::new_model_validation_error(
-            "At most one field must be marked as the id field with the `@id` attribute.",
-            &model.name,
-            ast_model.span,
-        ));
-
-        let multiple_id_criteria_error = Err(DatamodelError::new_model_validation_error(
-            "Each model must have at most one id criteria. You can't have `@id` and `@@id` at the same time.",
-            &model.name,
-            ast_model.span,
-        ));
```

Comment:
Are these errors changed now? (Can't locate the tests anymore)

Reply:
No, looks like they were already duplicated during Tom's refactoring and now trigger within the parser db

Reply:
Ah cool